cognite/neat/_session/_subset.py

```diff
@@ -80,3 +96,83 @@ def data_model(self, concepts: str | list[str]) -> IssueList:
             print(f"Subset to {after} concepts.")
 
         return issues
+
+    def core_data_model(self, concepts: str | list[str]) -> IssueList:
```

Comment:
What is the thinking about placing this method here and not under `neat.read.cdf.data_model.cognite_core()`? Feels strange to start a session with `neat.subset`.